package.json

```diff
@@ -9,8 +9,9 @@
     "fix:js": "npm run-script -- test:js --fix",
     "fix": "npm run-script fix:js && npm run-script fix:other",
     "prettier": "prettier \"**/*.{json,md,scss,yaml,yml}\"",
-    "test:other": "npm run-script -- prettier --list-different",
-    "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.{js,jsx}\""
+    "test:other": "npm run-script -- prettier --check -w",
+    "test:js": "eslint -v --ignore-path .gitignore --ignore-path .prettierignore \"**/*.{js,jsx}\"",
+    "prepare": "husky install"
```

Comment:
ah I see.. I installed husky a few times over with different versions.. one of those installations must've automatically inserted this built-in npm script (probably the latest version from the initial PR), so finally husky installed the proper pre-commit hook.

Reply:
Not sure I get what the "prepare" entry is for?

Reply:
it's a npm hook that'll be executed in certain npm commands: https://docs.npmjs.com/cli/v7/using-npm/scripts#prepare-and-prepublish. it's being called when installing (among other commands) and does `husky install`. this is what actually brings back the whole pre-commit logic. that didn't work before and seems to be the reason why the `test:js/other` targets weren't run in the past year or so.